### DISCUSSION TITLE
Module needs

### DISCUSSION THREAD

**Discussion by ppeyman** [breezefront/community#33] [5. General]
i installed breeze front for some of my customer and i have some question about swissuplabs modules 
Is necessary to  swissuplabs page speed when having a breeze theme?
Is it affects the results of Google Insights or not ?

regards

**Comment by vovayatsyuk** [comment]
No, it's not necessary. We do not use it on our [BreezeBlank](https://breeze.swissupdemo.com/breeze_blank/) and [BreezeEvolution](https://breeze.swissupdemo.com/breeze_evolution/) demos.

I don't have any numbers for the score with and without our module for BreezeThemes but I doubt it will increase noticeably. It depends on what score you have now, and what [google pagespeed tool](https://pagespeed.web.dev/) is telling you about your performance.

The main feature of the pagespeed module for breeze themes is webp images. If google insights recommend optimizing a lot of images - the module should increase the score. Maybe @dimdimich have some numbers?

**Reply by vovayatsyuk** [reply]
One important thing: webp images may not increase the score, but they will reduce the traffic for the store. This may save the hosting budget.
